x-pack/plugins/infra/public/components/logging/log_text_stream/item_field.tsx

```diff
@@ -19,5 +19,5 @@ export const LogTextStreamItemField = styled.div.attrs<{
       [switchProp.default]: props.theme.eui.euiFontSize,
     })};
   line-height: ${props => props.theme.eui.euiLineHeight};
-  padding: 2px ${props => props.theme.eui.euiSize};
+  padding: 2px ${props => props.theme.eui.euiSize} 2px 0;
```

Comment:
This leads to the timestamp being too close to the window edge:

![image](https://user-images.githubusercontent.com/973741/51971763-38db0a80-247a-11e9-82a5-fa7d76829f89.png)

Reply:
How about this...

![image](https://user-images.githubusercontent.com/41702/51997380-daab2900-2473-11e9-8e1e-9958c288d9bd.png)